tests/Data/Test_Transaction.cpp

```diff
@@ -42,6 +43,9 @@ BOOST_AUTO_TEST_CASE(test1)
 
     Address toAddr;
 
+    Mediator* m = nullptr;
+    ValidatorBase* m_validator = new Validator(*m);
```

Comment:
Even though this is a test case, don't use new. Many times when  the team is trying to detect memory leak, we will run valgrind against unit test. By doing so, valgrind will flag it out as leak, even though in reality, the core code isn't